lib/dock.rb

```diff
@@ -0,0 +1,58 @@
+class Dock
+  attr_reader :name, :max_rental_time
+
+  def initialize(name, max_rental_time)
+    @name = name
+    @max_rental_time = 3
+    @revenue = 0
+    @boats = {}
+
+  end
+
+  def rent(boat,renter)
+    @boats[boat] = renter
+  end
+
+  def log_hour
+    @boats.each do |boat, renter|
+        boat.add_hour
+      end
+  end
+
+  def return(boat)
```

Comment:
This method should probably remove the boat from the `@boats` hash. something like `@boats.delete(boat)`